File: bench.cpp
#include <iostream>

int main(int argc, char const *argv[]) {
  for(int i = 2; i < 100000; i++){
    bool prime = true;

    for(int j = 2; j*j <= i; j++){
        if(i % j == 0){
          prime = false;
          break;
        }
    }
    if(prime){
      std::cout << i << std::endl;

    }
  }

  return 0;
}

File: bench.rs

fn main() {
    for i in 2..100000 {
        let mut prime = true;
        for j in 2..i {
            if i % j == 0 {
                prime = false;
                break;
            }
        }
        if prime {
            println!("{}", i);
        }
    }
}
